lmfdb/belyi/main.py

```diff
@@ -710,6 +717,60 @@ def belyi_search(info, query):
         else:
             raise ValueError("%s is not a valid Belyi map label" % primitivization)
 
+passport_columns = SearchColumns([
+    LinkCol("plabel", "belyi.passport_label", "Label", url_for_belyi_passport_label),
+    MathCol("deg", "belyi.degree", "Degree"),
+    SearchCol("group", "belyi.group", "Group"),
+    MathCol("abc", "belyi.abc", "abc", align="left", short_title="abc triple"),
+    MathCol("lambdas", "belyi.ramification_type", "Ramification type", align="left"),
+    MathCol("g", "belyi.genus", "Genus"),
+    MathCol("pass_size", "belyi.pass_size", "Passport size"),
+    MathCol("num_orbits", "belyi.num_orbits", "Number of Galois orbits"),
+    MathCol("maxdegbf", "belyi.orbit_size", "Max orbit size"),
+    MathCol("triples", "belyi.permutation_triple", "Triples", align="left", default=False),
+    LinkCol("primitivization", "belyi.primitivization", "Primitivization", url_for_belyi_passport_label, default=False)
+])
+
+# passport search
+@search_wrap(
+table=db.belyi_passports,
+title="Passport search results",
+err_title="Passport search input error",
+columns=passport_columns,
+#shortcuts={"download": Subgroup_download()}, #TODO
+bread=lambda: get_bread([("Search Results", "")]),
+learnmore=learnmore_list,
+url_for_label=url_for_belyi_passport_label,
+)
+def passport_search(info, query={}):
+    parse_bracketed_posints(info, query, "abc_list", "a, b, c", maxlength=3)
+    query = query_convert_abc_list(query)
+    # a naive hack
+    if info.get("abc"):
+        for elt in ["a_s", "b_s", "c_s"]:
+            info_hack = {}
+            info_hack[elt] = info["abc"]
+            parse_ints(info_hack, query, elt)
+
+    parse_ints(info, query, "g", "g")
```

Comment:
It would be better to factor out the common functionality between this and `belyi_search` into a common function (like in [classical modular forms](https://github.com/LMFDB/lmfdb/blob/main/lmfdb/classical_modular_forms/main.py#L728))